src/core/Toast/Toaster.tsx

```diff
@@ -4,32 +4,19 @@
  *--------------------------------------------------------------------------------------------*/
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { getContainer } from '../utils/common';
 import { ToastCategory, ToastProps } from './Toast';
-import { ToastSettings, ToastMaster } from './ToastMaster';
+import { ToastMaster } from './ToastMaster';
+
+const TOASTS_CONTAINER_ID = 'iui-toasts-container';
 
 export type ToastOptions = Omit<
   ToastProps,
   'category' | 'isVisible' | 'id' | 'content'
 >;
 
 export default class Toaster {
-  private addToast: (settings: ToastSettings) => void;
-  private closeAllHandler: () => void;
-
-  constructor() {
-    this._addHandler = this._addHandler.bind(this);
-    this._closeAllHandler = this._closeAllHandler.bind(this);
-
-    const container = document.createElement('div');
-    document.body.appendChild(container);
-    ReactDOM.render(
-      <ToastMaster
-        addToastHandler={this._addHandler}
-        closeAllHandler={this._closeAllHandler}
-      />,
-      container,
-    );
-  }
+  private toasts: ToastProps[] = [];
```

Comment:
This doesn't need state?

Reply:
What do you mean? `useState` ?

Reply:
Oh it's not a react component, nevermind